app/assets/stylesheets/spotlight/_nestable.scss

```diff
@@ -36,9 +36,9 @@ tr.dd-item {
 
 .dd-handle { display: block; height: 100%; margin: 5px 0; padding: 5px 10px; color: #333; text-decoration: none; font-weight: bold; border: 1px solid #ccc;
     background: #fafafa;
-    background: -webkit-linear-gradient(top, #fafafa 0%, #eee 100%);
-    background:    -moz-linear-gradient(top, #fafafa 0%, #eee 100%);
-    background:         linear-gradient(top, #fafafa 0%, #eee 100%);
+    background: -webkit-linear-gradient(to bottom, #fafafa 0%, #eee 100%);
+    background:    -moz-linear-gradient(to bottom, #fafafa 0%, #eee 100%);
```

Comment:
Color literals like `#fafafa` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color literals like `#eee` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Colon after property should be followed by one space<br>Avoid vendor prefixes.